README.md

```diff
@@ -1,16 +1,22 @@
 # audio-guestbook
 The audio guestbook is a converted telephone handset that guests can use to leave recorded messages at weddings, parties and other events, as sold by companies such as "After the Tone", "Fête Fone", "Life on Record", "At the Beep", and others.
 
-Watch the full step-by-step tutorial on how to use the code here to build your own at https://youtu.be/dI6ielrP1SE
+Watch the full (but now slightly outdated) step-by-step tutorial on how to use the code here to build your own at https://youtu.be/dI6ielrP1SE
  
  ---
 ## Disabled MTP during recording
 In some cases, the MTP capability (allowing file transfer without removing the SD card) seems to interfere with recording. MTP medium-change checking is thus _disabled_ during recording. A suitable message is sent to the serial monitor so you can see when it's been enabled.
+
+---
+
 ## Audio tweaks
 ### Background
 The "modifications by h4yn0nnym0u5e, October 27th 2022" work significantly better if the audio library is "tweaked" to use larger data blocks and thus less-frequent audio updates. This allows more time for an SD card write before an interrupt is missed and audio data are lost.
+ 
 ### Installation method
-Unfortunately, "installation" is a bit messy, but you only have to do it once (unless you update your Arduino IDE or Teensyduino). The guestbook sketch folder contains two Arduino architecture configuration files, `boards.local.txt` and`platform.txt`. These must be used to update your existing configuration files; in Windows systems these will typically be in `C:\Program Files (x86)\Arduino\hardware\teensy\avr` [todo: where are they on Linux and Mac?].
+Unfortunately, "installation" is a bit messy, but you only have to do it once (unless you update your Arduino IDE or Teensyduino).
+#### Tools menu option
+ The guestbook sketch folder contains two Arduino architecture configuration files, `boards.local.txt` and`platform.txt`. These must be used to update your existing configuration files; in Windows systems these will typically be in `C:\Program Files (x86)\Arduino\hardware\teensy\avr` [todo: where are they on Linux and Mac?].
```

Comment:
Hey @h4yn0nnym0u5e  I was going to make a small update to the README to include the config path on Macs and saw your PR so I thought I could just suggest a change instead of make my own PR.

```suggestion
 The guestbook sketch folder contains two Arduino architecture configuration files, `boards.local.txt` and`platform.txt`. These must be used to update your existing configuration files; in Windows systems these will typically be in `C:\Program Files (x86)\Arduino\hardware\teensy\avr` or `/Applications/Teensyduino.app/Contents/Java/hardware/teensy/avr` on Mac [todo: where are they on Linux?].
```

Reply:
How do I fix this error code on a mac? I am trying my best to understand it.


" 
Arduino: 1.8.19 (Mac OS X), TD: 1.57, Board: "Teensy 4.0, MTP Disk (Experimental), 600 MHz, Faster, US English"











Multiple libraries were found for "SD.h"
 Used: /private/var/folders/_2/tl13lpw51jj5_y71zm7qd__40000gp/T/AppTranslocation/3E053CC5-9134-4F53-AC58-809E7713ABB8/d/Teensyduino.app/Contents/Java/hardware/teensy/avr/libraries/SD
 Not used: /private/var/folders/_2/tl13lpw51jj5_y71zm7qd__40000gp/T/AppTranslocation/3E053CC5-9134-4F53-AC58-809E7713ABB8/d/Teensyduino.app/Contents/Java/libraries/SD
/Users/odastaff/Desktop/Audiobook/Audiobook.ino:30:24: fatal error: MTP_Teensy.h: No such file or directory
compilation terminated.
Error compiling for board Teensy 4.0.


This report would have more information with
"Show verbose output during compilation"
option enabled in File -> Preferences.

Reply:
@DaT729 , that's not an issue relating to mac, but to a missing library (people on Windows also get this error). The issue and its solution can be found on the "issues" page of this project: https://github.com/playfultechnology/audio-guestbook/issues/63